docs/language/contracts.mdx

```diff
@@ -387,6 +387,29 @@ let signer: AuthAccount = ...
 let contract = signer.contracts.get(name: "Test")
 ```
 
+### Borrowing a Deployed Contract
+
+In contrast to a static contract import `import T from 0x1`, 
+which will always perform an import of a type, 
+contracts can be "borrowed" to effectively perform a dynamic import dependent on a specific execution path. 
+
+A reference to a deployed contract contract can obtained using the `borrow` function:
+
+  ```cadence
+  fun borrow<T: &Any>(name: String): T?
+  ```
+
+  This returns a reference to the contract value stored with that name on the account, 
+  if it exists, and if it has the provided type `T`. 
```

Comment:
I assume it is possible to just provide the name of the contract as the type if it doesn't conform to an interface, right? It would be good to clarify that here